core-web/libs/dotcms-scss/shared/_colors.scss

```diff
@@ -104,8 +104,8 @@ $color-accessible-text-green-bg: $color-alert-green-light;
 $color-accessible-text-purple: $color-palette-secondary-600;
 $color-accessible-text-purple-bg: $color-palette-secondary-100;
 
-$color-accessible-text-blue: $color-palette-primary-500;
-$color-accessible-text-blue-bg: $color-palette-primary-200;
+$color-accessible-text-blue: $color-palette-primary-600;
```

Comment:
What is is the diff between alert and this?

Reply:
Removed the alert one.